src/components/tipRecords/YouTubeEmbed.vue

```diff
@@ -1,73 +1,85 @@
 <template>
   <div
-    class="tip__cover-preview"
+    class="youtube-embed"
   >
-    <iframe
-      v-if="isPlaying"
-      :src="`https://www.youtube-nocookie.com/embed/${videoId}?autoplay=1&rel=0`"
-      frameborder="0"
-      allow="accelerometer;
+    <CookiesDialog
+      v-if="showCookiesDialog && !isAllowed"
+      :address="address"
+      :scope="'YouTube'"
+    />
+    <div class="tip__cover-preview">
```

Comment:
```suggestion
    <div
      v-else
      class="tip__cover-preview"
    >
```
and to remove the useless wrapping node (`youtube-embed` class)

Reply:
So, it can't be done because of the designs.